launchpad_backend/launchpad_server/routes/index.py

```diff
@@ -13,11 +17,38 @@
 
 bcrypt = Bcrypt(app)
 
+def setup_db():
+    for data in startup_data:
+        collection = mongo.db[data["tableName"]] # similar to table in relational db
+        collection.drop()
+        insertList = []
+
+        for record in data["records"]: # similar to row in relational db
+            insertList.append(record)
+
+        collection.insert_many(insertList)
+        collection.create_index([(data["index"], pymongo.ASCENDING)], unique=True) # Enforces primary key like restrcitions
+
+setup_db() # Resets db to startup_data everytime backend is saved or ran
+
+
+@app.route("/applications")
+def check_db():
```

Comment:
`check_db()` is already defined under the `/check_db` route so change this function name to something else.

When I try to run this I get the following error:
`AssertionError: View function mapping is overwriting an existing endpoint function: check_db`